NEWS.md

```diff
@@ -0,0 +1,10 @@
+SelfControlledCohort 1.6.0
+==========================
+- Added changelog in NEWS.md
```

Comment:
Please distinguish between 'Changes' and 'Bug fixes'. (See for example [here](https://github.com/OHDSI/Andromeda/blob/master/NEWS.md))